### ISSUE TITLE
Filter for discussion titles

### ISSUE THREAD

**Issue by michaeltaidev** [FriendsOfFlarum/filter#27]
This extension would be perfect if the filters also applied to discussion titles. I gotta tell ya, these spammers are getting quite crafty with their workarounds.

**Comment by ShoKo-oo** [comment]
> This extension would be perfect if the filters also applied to discussion titles. I gotta tell ya, these spammers are getting quite crafty with their workarounds.

I agree with your statement that I am troubled by spam messages every day
